staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -751,11 +747,11 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 			getResp.Kvs[i] = nil
 		}
 
-		// no more results remain or we didn't request paging
-		if !hasMore || !paging {
+		// no more results remain
+		if !hasMore {
 			break
 		}
-		// we're paging but we have filled our bucket
```

Comment:
comment still looks relevant, paging is conditional on the request, not just the storage support

Reply:
done